### PR TITLE
[travis][appveyor] use symfony/flex to accelerate builds

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Playing with https://github.com/symfony/flex/pull/409

The optimization is required because appveyor is transiently failing with OOM errors, see e.g.
https://ci.appveyor.com/project/fabpot/symfony/build/1.0.39377

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -188,6 +188,16 @@ install:
           SYMFONY_VERSION=$(cat composer.json | grep '^ *"dev-master". *"[1-9]' | grep -o '[0-9.]*')
       fi
 
+    - |
+      # Install symfony/flex
+      if [[ $deps = low ]]; then
+          export SYMFONY_REQUIRE='>=2.3'
```
Comment (nicolas-grekas) [inline .travis.yml]: should be bumped to `>=2.8` on 3.4 and `>=3.4` on 4.1

Comment (stof) [reply .travis.yml]: that looks wrong to me. Forcing `>=3.4` for deps=low on 4.1 might hide the fact that the actual low deps can be older than that based on the requirements for instance..

Comment (nicolas-grekas) [reply .travis.yml]: You're right, that happens on deps=low for 4.1.
Let's bump to `>=3.4` when 2.8 is EOLed then, would that be OK for you?

Comment (stof) [reply .travis.yml]: even then, it might hide the fact that our low dep is wrong

Comment (nicolas-grekas) [reply .travis.yml]: Fine, let's keep it to 2.3 for now...

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**